cmd/create/ocmrole/cmd.go

```diff
@@ -499,3 +508,24 @@ func createPermissionPolicy(reporter *rprtr.Object, awsClient aws.Client, policy
 
 	return nil
 }
+
+func checkAdminRoleExists(reporter *rprtr.Object, awsClient aws.Client, roleName string, isAdmin bool) error {
+	exists, _, err := awsClient.CheckRoleExists(roleName)
+	if err != nil {
+		return err
+	}
+	if exists {
+		reporter.Warnf("Role '%s' already exists", roleName)
+		isExistingRoleAdmin, err := awsClient.IsAdminRole(roleName)
+		if err != nil {
+			return err
+		}
+		if !isAdmin && isExistingRoleAdmin {
```

Comment:
This is a problem. You will return this error to the user but won't exit

Reply:
I did it on purpose because it is for manual mode, and the user doesn't actually try to create a role yet.
I wanted only to warn the user and continue and display the commands to create roles.
WDYT?

Reply:
I think it should exit. That was the intention of the card. If the user gets the commands and run them, they would fail, and that is what we want to prevent

Reply:
Per @vkareh 's [comment](https://issues.redhat.com/browse/SDA-5140?focusedCommentId=19428457&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-19428457
) I mislead you and you should warn and continue. 

@vkareh can you confirm?

Reply:
Correct. This is for manual mode only, so printing a warning before the commands makes sense. The user can decide how to proceed after that.